### ISSUE TITLE
alive?

### ISSUE THREAD

**Issue by dynamix** [ghempton/ember-script#52]
Quick question - is this project still alive? We wrote some stuff in Emberscript and it would be sad if we have to throw it away again.

**Comment by tygriffin** [comment]
+1

**Comment by cbornhoftssc** [comment]
Been 9 months without any sort of activity. Think it's safe to say this is dead :(

**Comment by andrewkatz** [comment]
@ghempton Any word on this?

**Comment by zekefast** [comment]
Is it dead? :(

**Comment by ghempton** [comment]
I am personally pretty low on bandwidth for the project :( so I don't see myself adding any new features any time soon. We still have a large project that uses EmberScript in its current form, however.

That said, if anyone wants to step up and take over the reigns, I would definitely try and help with that process as much as possible.

**Comment by edborden** [comment]
Biggest issue right now is getting all of the build tools to work correctly with Ember-CLI and newest Ember versions...

**Comment by zekefast** [comment]
Yes, I decided to step out from emberscript while my code base resonably small. Nevertheless I liked the attempt.

One thing which simple to do is to upgrade it to be compatible with node.js `0.12.0`. But I think it worth a separation issue.

**Comment by clov3r** [comment]
This is working for me with ember 0.11, cli 0.2.3. I'm interested in helping out @ghempton

**Comment by clov3r** [comment]
Perhaps with @edborden and @kolybasov, we'd be able to keep this project alive. There are a few outstanding issues, but it seems this project is working, so let's keep it so :)

**Comment by edborden** [comment]
I've been using github.com/kolybasov/broccoli-ember-script, which works for me for the moment with ember 0.11, cli 0.2.3. I moved over to that repo due to https://github.com/ghempton/ember-script/issues/53

**Comment by clov3r** [comment]
:+1:

**Comment by auvipy** [comment]
is this project work with latest ember versions? If not I want to contribute to move this forward. any kind of guide would be great

**Comment by v3ss0n** [comment]
seems dead to me , nobody at ember community using it too.

**Comment by auvipy** [comment]
only livescript is real then

**Comment by ghempton** [comment]
I no longer have time to contribute to the project and we no longer use Ember... definitely room for a new maintainer.

**Comment by auvipy** [comment]
this should be moved under ember organization

**Comment by v3ss0n** [comment]
I guess nobody uses it actually.
